src/nvim/generators/gen_unicode_tables.lua

```diff
@@ -252,7 +252,8 @@ local build_emoji_table = function(ut_fp, emojiprops, doublewidth, ambiwidth)
 
       -- Characters below 1F000 may be considered single width traditionally,
       -- making them double width causes problems.
-      if n >= 0x1f000 then
+      -- Make an exception for the Regional Indicator Symbols 0x1F1E6-0X1F1FF
```

Comment:
this comment is worded poorly, given the comment just above it. Please take care to make the comments work together for the reader's understanding.